python_todo/views.py

```diff
@@ -0,0 +1,44 @@
+from flask import Flask, jsonify, request, render_template
```

Comment:
You should definitely write tests for this. Learning how to test and what to test is the most perhaps the most important skill. It might sound like a waste of time at first, but it's not. Believe me.